packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -1029,3 +1074,28 @@ class ScrollAction extends Action<ScrollIntent> {
     );
   }
 }
+
+// Not using a RestorableDouble because we want to allow null values and override
+// [enabled].
+class _RestorableScrollOffset extends RestorableValue<double> {
+  @override
+  double createDefaultValue() => null;
+
+  @override
+  void didUpdateValue(double oldValue) {
+    notifyListeners();
```

Comment:
even if it didn't change?

Reply:
RestorableValue only calls didUpdateValue if old and new value are different.